src/staticdata.c

```diff
@@ -643,6 +643,7 @@ static void jl_queue_module_for_serialization(jl_serializer_state *s, jl_module_
 // be the "source" rather than merely a cross-reference.
 static void jl_insert_into_serialization_queue(jl_serializer_state *s, jl_value_t *v, int recursive)
 {
+    void **bp;
```

Comment:
Initialize with NULL as a best practice?

Reply:
Good idea. It was split from its initialization only because C doesn't like a goto label that points at a declaration.